modules/installation-cloudformation-subnet-localzone.adoc

```diff
@@ -14,53 +14,6 @@ you need for your {product-title} cluster that uses AWS Local Zones.
 ====
 [source,yaml]
 ----
-# CloudFormation template used to create Local Zone subnets and dependencies
-AWSTemplateFormatVersion: 2010-09-09
-Description: Template for create Public Local Zone subnets
-
-Parameters:
-  VpcId:
-    Description: VPC Id
-    Type: String
-  ZoneName:
-    Description: Local Zone Name (Example us-east-1-nyc-1a)
-    Type: String
-  SubnetName:
-    Description: Local Zone Name (Example cluster-public-us-east-1-nyc-1a)
-    Type: String
-  PublicRouteTableId:
-    Description: Public Route Table ID to associate the Local Zone subnet
-    Type: String
-  PublicSubnetCidr:
-    AllowedPattern: ^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(1[6-9]|2[0-4]))$
-    ConstraintDescription: CIDR block parameter must be in the form x.x.x.x/16-24.
-    Default: 10.0.128.0/20
-    Description: CIDR block for Public Subnet
-    Type: String
-
-Resources:
-  PublicSubnet:
-    Type: "AWS::EC2::Subnet"
-    Properties:
-      VpcId: !Ref VpcId
-      CidrBlock: !Ref PublicSubnetCidr
-      AvailabilityZone: !Ref ZoneName
-      Tags:
-      - Key: Name
-        Value: !Ref SubnetName
-      - Key: kubernetes.io/cluster/unmanaged
-        Value: "true"
-
-  PublicSubnetRouteTableAssociation:
-    Type: "AWS::EC2::SubnetRouteTableAssociation"
-    Properties:
-      SubnetId: !Ref PublicSubnet
-      RouteTableId: !Ref PublicRouteTableId
-
-Outputs:
-  PublicSubnetIds:
-    Description: Subnet IDs of the public subnets.
-    Value:
-      !Join ["", [!Ref PublicSubnet]]
+include::https://raw.githubusercontent.com/openshift/installer/master/upi/aws/cloudformation/01.99_net_local-zone.yaml[]
```

Comment:
We updated this in the day-0 docs: https://63729--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-localzone#installation-cloudformation-subnet-localzone_installing-aws-localzone